.github/workflows/build-verification.yml

```diff
@@ -20,3 +20,40 @@ jobs:
           develocity-access-key: ${{ secrets.DV_SOLUTIONS_ACCESS_KEY }}
       - name: Build with Gradle
         run: ./gradlew build -x signPluginMavenPublication -i -Porg.gradle.java.installations.auto-download=false
+
+  selftest:
+    name: Publish to Maven Local and Test with Locally Published Plugin
+    runs-on: ubuntu-latest
+    needs: verification
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Set up JDK 21
+        uses: actions/setup-java@v4
+        with:
+          java-version: '21'
+          distribution: 'temurin'
+      - name: Set up Gradle
+        uses: gradle/actions/setup-gradle@v4
+        with:
+          develocity-access-key: ${{ secrets.DV_SOLUTIONS_ACCESS_KEY }}
+      - name: Disable signing on CI
+        run: sed -i 's/isRequired = providers.environmentVariable("CI").isPresent/isRequired = false/' build.gradle.kts
```

Comment:
I would avoid editing build files like this.

Instead, I recommend including a check for publishToMavenLocal as a requested task in the signing condition.

Adding a system property to disable signing if included on the command line would also be a good option.